packages/manager/apps/web/client/app/hosting/cdn/order/hosting-cdn-order.html

```diff
@@ -15,5 +21,62 @@
     user=":: $ctrl.user"
     on-error=":: $ctrl.onError(error)"
     on-success=":: $ctrl.onSuccess($ctrl.workflowOptions.serviceId ? result : checkout)"
+    send-current-state="$ctrl.getOrderState(state)"
 >
+    <oui-step-form
+        header="{{:: 'hosting_cdn_order_choose_plan' | translate }}"
+        prevent-next="true"
```

Comment:
no need to go to the next step?

Reply:
`prevent-next` makes the step requiring validation to go to next step. `valid` property is used to describe the condition :)